web/server/system/shell.go

```diff
@@ -33,7 +35,7 @@ func (self *Shell) GoTest(directory, packageName string, tags, arguments []strin
 	reportPath := filepath.Join(self.reportsPath, reportFilename)
 	reportData := reportPath + ".txt"
 	reportHTML := reportPath + ".html"
-	tagsArg := "-tags=" + strings.Join(tags, ",")
+	tagsArg := "-tags=" + strings.Join(append(self.gotags, tags...), " ")
```

Comment:
@riannucci this is actually an insidious little flag in my opinion. The original seems incorrect. [See here for more info](https://github.com/golang/go/issues/8772)